main/src/cgeo/geocaching/settings/ViewSettingsActivity.java

```diff
@@ -118,8 +118,8 @@ public View getView(final int position, final View convertView, @NonNull final V
             View v = convertView;
             if (null == convertView) {
                 v = getLayoutInflater().inflate(R.layout.twotexts_twobuttons_item, parent, false);
-                ((ImageButton) v.findViewById(R.id.button_left)).setImageResource(R.drawable.ic_menu_edit);
-                ((ImageButton) v.findViewById(R.id.button_right)).setImageResource(R.drawable.ic_menu_delete);
+                ((Button) v.findViewById(R.id.button_left)).setCompoundDrawablesRelativeWithIntrinsicBounds(R.drawable.ic_menu_edit, 0, 0, 0);
```

Comment:
Thanks for stepping into this issue! 

I read somewhere that there's a method `Button.setIconResource(R.drawable.ic_whatever)`. Don't know if it's part of the Button or MaterialButton class. However every button can simply be casted to MaterialButton inside the Java code... Maybe that solves the issue. 

If not there's a style `Widget.MaterialComponents.Button.OutlinedButton.Icon` from which our icon button style should be inherited from.

>In order to get the correct icon padding, it is recommended that you use a *.Icon style variant

Reply:
Thanks, Casting to MaterialButton worked, but unfortunately not for the clear-icon in filter-view (AttributesFilterViewHolder) 😲
Inherit icon style from `Widget.MaterialComponents.Button.OutlinedButton.Icon` does not work either. 

Other ideas?

Reply:
> Thanks, Casting to MaterialButton worked, but unfortunately not for the clear-icon in filter-view (AttributesFilterViewHolder) 😲
> Inherit icon style from `Widget.MaterialComponents.Button.OutlinedButton.Icon` does not work either.
> 
> Other ideas?

I assume the difference between the button in AttributesFilterViewHolder and all others is that the button in `AttributesFilterViewHolder` is created via code, right?

I had styling problems with material.io before when creating view elements programatically. I solved them by creating simple layout files just containing one element. For example, I created layout `button_view.xml` which just contains a base-formatted `MaterialButton`. When I create the button by inflating this layout programatically, then later layouting seems to work correctly (and applies also other material styles correctly).

I assume `AttributesFilterViewHolder` simply needs to be changed to inflate the button in question out of this layout file or a similar one for Icon buttons.

Reply:
> I assume the difference between the button in AttributesFilterViewHolder and all others is that the button in `AttributesFilterViewHolder` is created via code, right?

Right 

> I assume `AttributesFilterViewHolder` simply needs to be changed to inflate the button in question out of this layout file or a similar one for Icon buttons.

Thanks, I will try that

Reply:
Great! Worked!
![image](https://user-images.githubusercontent.com/70113341/122227437-27fa8a80-ceb7-11eb-897b-c5a7d39d4fb2.png)

Thanks @eddiemuc

Reply:
Looks good, thanks for your work 👍🎉

The PR is currently still set to draft, anything which needs to be done or is it ready?